PowerVCF.psm1

```diff
@@ -1128,18 +1119,17 @@ Function Set-VCFCluster {
         if ( -not $PsBoundParameters.ContainsKey("json") -and ( -not $PsBoundParameters.ContainsKey("markForDeletion"))) {
             Throw "You must include either -json or -markForDeletion"
         }
-        if ($PsBoundParameters.ContainsKey("json")) {
-            validateJsonInput # Calls validateJsonInput Function to check the JSON file provided exists
-            # Validate the provided JSON input specification file
-            $response = Validate-VCFUpdateClusterSpec -clusterid $id -json $ConfigJson
+        
+            $jsonBody = validateJsonInput -json $json   # validate input file and format
+            $response = Validate-VCFUpdateClusterSpec -clusterid $id -json $jsonBody # validate the JSON provided meets the cluster specifications format
             # the validation API does not currently support polling with a task ID
```

Comment:
Worth checking if this is still the case!

Reply:
hey @LifeOfBrianOC are you referring to the polling ID from the validation API ?

Reply:
I checked and the validation on 4.4.1 still works the same. so from my perspective no changes are required here.